src/quick-start.md

```diff
@@ -99,54 +113,37 @@ spec:
-> registered namespaced resources on its own namespace.
-
-## Enforce your first policy
-
-We will use the [`pod-privileged` policy](https://github.com/kubewarden/pod-privileged-policy).
-This policy prevents the creation of privileged containers inside of a Kubernetes cluster.
+Overview of the attributes of the `ClusterAdmissionPolicy` resource:
+
+| Required | Placeholder         | Description    |
+|:--------:| ------------------- | ----------------------------- |
+| - | `policy-server`  | identifies an existing `PolicyServer` object. The policy will be served only by this `PolicyServer` instance. A `ClusterAdmissionPolicy` that doesn't have an explicit `PolicyServer`, will be served by the one named `default`. |
+| ✅ | `module`  | The location of the Kubewarden policy. The following options are allowed:  |
+| | | - `registry`: The policy is downloaded from an [OCI artifacts](https://github.com/opencontainers/artifacts) compliant container registry |
+| | | - `http`, `https`: The policy is downloaded from a regular HTTP(s) server |
+| | | - `file`: The policy is loaded from a file in the computer filesystem |
+| ✅ | `resources` | The Kubernetes resources evaluated by the policy |
+| ✅ | `operations` | what operations for the previously given types should be forwarded to this admission policy by the API server for evaluation. |
+| ✅ | `mutating` | a boolean value that must be set to `true` for policies that can mutate incoming requests |
+| - | `settings` | The list of the policy configuration values |
+| - | `failurePolicy` | The action to take if the policy  unrecognized or timeout error. The following options are allowed: |
+| | | - `Ignore`: an error calling the webhook is ignored and the API request is allowed to continue |
+| | | - `Fail`: an error calling the webhook causes the admission to fail and the API request to be rejected |
+
+The complete documentation of this Custom Resource can be found [here](https://github.com/kubewarden/kubewarden-controller/blob/main/docs/crds/README.asciidoc) or on [docs.crds.dev](https://doc.crds.dev/github.com/kubewarden/kubewarden-controller).
+
+> **NOTE:** The  `ClusterAdmissionPolicy` resources are registered with a `*` webhook `scope`, which means that registered webhooks will be forwarded all requests matching the given `resources` and `operations` -- either namespaced (in any namespace), or cluster-wide resources.
+
+> **NOTE:** The `ClusterAdmissionPolicy` resource is cluster-wide. There are plans to also provide a namespaced version that will only impact registered namespaced resources on its own namespace.
+
+## Example: Enforce your first policy
+
+For this first example, we will use the [`pod-privileged` policy](https://github.com/kubewarden/pod-privileged-policy).
+Our goal will be to prevent the creation of privileged containers inside our Kubernetes cluster by enforcing this policy.
 
 Let's define a `ClusterAdmissionPolicy` for that:
 
-```shell
+```console
```

Comment:
Not related to your change. We just renamed some of our PSP policies.

Can you replace `module: registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.5` with `module: registry://ghcr.io/kubewarden/policies/privileged-psp:v0.1.9`?

Reply:
Updated policy reference